src/app/use-cases/validate-document.ts

```diff
@@ -67,13 +65,9 @@ export const makeValidateDocument =
         pipe(
           request,
           addUrlToDocument(payload.documentId, payload.documentUrl),
-          E.map((documents) => ({ ...request, documents }))
+          E.map((documents) => ({ ...request, documents })),
+          E.chain(dispatch("UPLOAD_DOCUMENT"))
         )
       ),
-      TE.chain(upsertSignatureRequest),
-      TE.chain((signatureRequest) =>
-        status(signatureRequest) === "WAIT_FOR_SIGNATURE"
-          ? enqueueRequestAwaitingSignature(signatureRequest)
-          : TE.right(signatureRequest)
-      )
+      TE.chain(upsertSignatureRequest)
```

Comment:
it looks like `enqueueRequestAwaitingSignature` gets lost, is this on purpose?

Reply:
Yes, `enqueueRequestAwaitingSignature` was used to send the message right away. Later this part will still be reviewed with another PR to send the documents for validation.